demo/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -37,8 +39,10 @@ const MeetingForm: React.FC = () => {
   const [nameErr, setNameErr] = useState(false);
   const [region, setRegion] = useState(appRegion);
   const [isLoading, setIsLoading] = useState(false);
+  const [isSelected, setIsSelected] = useState(false)
```

Comment:
nit: Can we rename this to something that says what it actually does? IsSpectatorModeSelected or something

Reply:
Yes, will do!